exercises/practice/clock/.meta/config.json

```diff
@@ -0,0 +1,20 @@
+{
+  "blurb": "Convert a long phrase to its acronym",
+  "authors": [
+    "marksiemers"
+  ],
+  "contributors": [],
+  "files": {
+    "solution": [
+      "src/clock.cr"
+    ],
+    "test": [
+      "spec/clock_spec.cr"
+    ],
+    "example": [
+      ".meta/src/example.cr"
```

Comment:
The `solution` and the `example` are the exact same code - I wasn't sure if this was desired. If it should be something different, please let me know.